src/ROUTES.ts

```diff
@@ -7,149 +7,168 @@ import CONST from './CONST';
 
 // prettier-ignore
 export default {
-    HOME:                                       '',
```

Comment:
Why was all the whitespace removed from this file? It was very intentional and this file is now very unreadable.